resolver-dns/src/main/java/io/netty/resolver/dns/DnsCache.java

```diff
@@ -45,24 +46,27 @@
     /**
      * Return the cached entries for the given hostname.
      * @param hostname the hostname
+     * @param additionals the additional records
      * @return the cached entries
      */
-    List<DnsCacheEntry> get(String hostname);
+    List<DnsCacheEntry> get(String hostname, DnsRecord[] additionals);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Consider using varargs for methods or constructors which take an array the last parameter. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://garage.netty.io/sonarqube/coding_rules#rule_key=pmd%3AUseVarargs)